t/ui/17-product-log.t

```diff
@@ -47,19 +48,18 @@ if (!$driver) {
     exit(0);
 }
 
-my $t = Test::Mojo->new('OpenQA::WebAPI');
-
-# we need to talk to the phantom instance or else we're using wrong database
-my $url = 'http://localhost:' . OpenQA::SeleniumTest::get_mojoport;
-
-# Schedule iso - need UA change to add security headers
-# XXX: Test::Mojo loses it's app when setting a new ua
-# https://github.com/kraih/mojo/issues/598
+# setup test application with API access
+# note: Test::Mojo loses it's app when setting a new ua (see https://github.com/kraih/mojo/issues/598).
```

Comment:
Nit-pick: might as well make it `its app` if you're touching that comment 😉️

Reply:
ok